cellprofiler/gui/module_view/_module_view.py

```diff
@@ -1822,6 +1822,17 @@ def make_measurement_control(self, v, panel):
         # * panel = None - create the controls
         # * panel != None - find the controls
         #
+        category = v.get_category(self.__pipeline)
```

Comment:
Out of the scope for the related issue, but `v` is an item from the list returned by `__module.visible_settings()`, so I assume `v` means `visible_setting`? If yes, can we change it to that explicitly?

Reply:
Fine with me, though I can neither confirm nor deny that that's what v is